.github/workflows/ci.yml

```diff
@@ -110,6 +110,22 @@ jobs:
             nodejs_version: "^22.9.0"
             nodejs_version_major: 22
             platform: win32-x64
+          - os: windows-11-arm
+            nodejs_arch: arm64
+            nodejs_version: "^18.17.0"
+            nodejs_version_major: 18
+            platform: win32-arm64
+            prebuild: true
```

Comment:
Windows arm64 support was only added in Node v20, see https://github.com/nodejs/node/pull/47233.